projects/pgai/pgai/alembic/operations.py

```diff
@@ -22,18 +22,20 @@ def create_vectorizer(cls, operations: Operations, **kw: Any):
 
 
 class DropVectorizerOp(MigrateOperation):
-    def __init__(self, vectorizer_id: int | None, drop_all: bool):
+    def __init__(self, vectorizer_id: int | None, name: str | None, drop_all: bool):
```

Comment:
Nit: for consistency
```suggestion
    def __init__(self, vectorizer_id: int | None, vectorizer_name: str | None, drop_all: bool):
```
Or rather change the `vectorizer_id` to just `id`.
Same applies to the rest of the code